src/Redux/PollListSlice.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const base_URL = process.env.REACT_APP_BASE_URL;
+export const fetchPollList = createAsyncThunk(
+  "fetchPollList",
+  async (number: number) => {
+    const accessToken: any = localStorage.getItem("userToken");
```

Comment:
why any? local storage only support strings , so it can string @Rishab-innovative